src/messages/variant.rs

```diff
@@ -90,6 +101,14 @@ impl Debug for Variant {
             Self::ParsecRequest(version, _) => write!(f, "ParsecRequest({}, ..)", version),
             Self::ParsecResponse(version, _) => write!(f, "ParsecResponse({}, ..)", version),
             Self::Ping => write!(f, "Ping"),
+            Self::Bounce {
+                elders_version,
+                message,
+            } => f
+                .debug_struct("Bounce")
+                .field("elders_version", elders_version)
+                .field("message", &HexFmt(message))
```

Comment:
Space is going to be the same (`HexFmt` truncates the message) but using message hash is probably more useful as it can be looked up in other parts of the log to find where the message originated. Will change it.